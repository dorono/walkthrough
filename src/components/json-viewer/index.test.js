import React from 'react';
import {shallow} from 'enzyme';
import JsonViewer from './index';

const defaultProps = {
    data: JSON.stringify({test: ''}),
};

describe('JsonViewer', () => {
    it('should render without errors', () => {
        shallow(<JsonViewer {...defaultProps} />);
    });

    it('should match snapshot', () => {
        expect(shallow(<JsonViewer {...defaultProps} />)).toMatchSnapshot();
    });
});
