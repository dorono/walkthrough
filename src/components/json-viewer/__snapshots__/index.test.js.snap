// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JsonViewer should match snapshot 1`] = `
<pre
  className="root"
>
  <code
    className="json"
    dangerouslySetInnerHTML={
      Object {
        "__html": "
<span class=\\"line\\">{</span>
<span class=\\"line\\">    <span class=\\"key\\">\\"test\\":</span> <span class=\\"string\\">\\"\\"</span></span>
<span class=\\"line\\">}</span>",
      }
    }
  />
</pre>
`;
